src/tenant.ts

```diff
@@ -65,15 +65,16 @@ export class Tenant {
         );
     }
 
-    modifyTenant(passwd: string, extra?: object) {
+    modifyTenant(active: boolean, status: string, metadata = {}) {
```

Comment:
all of them are non mandatory so please make them optional.

Reply:
made the required changes